calebwagner/python-server
# ch 9 completed, getting all lists from db

#### Changes Made
1. Modified file `employees` to include `sqlite3` import, `json` import, `models from lists`, and query for all lists.
1. Modified file `locations` to include `sqlite3` import, `json` import, `models from lists`, and query for all lists.
1. Modified file `animals` to include `sqlite3` import, `json` import, `models from lists`, and query for all lists.
1. Modified file `customers` to include `sqlite3` import, `json` import, `models from lists`, and query for all lists.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-ch9
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > check for above changes in code. 
    > in Postman ...
-  type in: http://localhost:8088/locations, http://localhost:8088/employees, http://localhost:8088/customers, http://localhost:8088/animals & use GET method to get all lists.

4. View code file.
    > Confirm file modifications are present as indicated above.
